decidim-bulletin-board-app/app/commands/process_key_ceremony_step.rb

```diff
@@ -69,12 +70,16 @@ def process_message
   def create_response_log_entry!
     return unless response_message
 
-    LogEntry.create!(
+    log_entry = LogEntry.create!(
       election: election,
       signed_data: BulletinBoard.sign(response_message),
       log_type: :key_ceremony,
       bulletin_board: true
     )
+
+    LogEntryNotifier.new(log_entry).notify_subscribers
```

Comment:
again, maybe we can store this log entry as `response_log_entry` and wait after saving all the models to notify the users

Reply:
Yeah, I'll do that! 👍